applications/MultilevelMonteCarloLaplacianApplication/python_scripts/multilevel_montecarlo_laplacian_analysis.py

```diff
@@ -0,0 +1,724 @@
+from __future__ import absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics as Kratos
+# KratosMultiphysics.Logger.GetDefaultOutput().SetSeverity(KratosMultiphysics.Logger.Severity.WARNING) # avoid printing of Kratos things
+Kratos.Logger.GetDefaultOutput().SetSeverity(Kratos.Logger.Severity.WARNING)
+import KratosMultiphysics.MyStochasticLaplacianApplication as Poisson
+
+from analysis_stage import AnalysisStage
+
+import numpy as np
+import time
+
+from pycompss.api.task import task
+from pycompss.api.api import compss_wait_on
+from pycompss.api.parameter import *
+
+class MultilevelMonteCarloLaplacianAnalysis(AnalysisStage):
+    """Main script for MultilevelMonte Carlo Poisson simulations using the pure_diffusion solver"""
+
+
+    def __init__(self,model,parameters,sample):
+        self.sample = sample
+        super(MultilevelMonteCarloLaplacianAnalysis,self).__init__(model,parameters)
+
+            
+    def _CreateSolver(self):
+        import pure_diffusion_solver
+        solver = pure_diffusion_solver.CreateSolver(self.model,self.project_parameters["solver_settings"])
+        self.PureDiffusionSolver = solver
+        return self.PureDiffusionSolver
+
+    
+    def _GetSimulationName(self):
+        return "Multilevel Monte Carlo Poisson Analysis"
+
+    
+    def ApplyBoundaryConditions(self):
+        super(MultilevelMonteCarloLaplacianAnalysis,self).ApplyBoundaryConditions()
+        ## define the forcing function
+        for node in self.model.GetModelPart("MLMCLaplacianModelPart").Nodes:
+            coord_x = node.X
+            coord_y = node.Y
+            # forcing = -432.0 * coord_x * (coord_x - 1) * coord_y * (coord_y - 1)
+            forcing = -432.0 * (coord_x**2 + coord_y**2 - coord_x - coord_y)
+            node.SetSolutionStepValue(Poisson.FORCING,forcing*self.sample)
```

Comment:
here you should use the variable that is used for this purpose in the convection diffusion application (i think it is HEAT_FLUX, but you'll need to verify this)